services/david/david.tester.js

```diff
@@ -37,6 +37,13 @@ t.create('david peer dependencies (valid)')
     value: isDependencyStatus
   }));
 
+t.create('david dependencies (path)')
+  .get('/babel/babel.json?path=packages/babel-core')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'dependencies',
+    value: isDependencyStatus
+  }));
+
```

Comment:
Could you also add a test for when `?path` is an invalid path.

Reply:
Will do.